src/components/dashboard/inventory/InventoryInput.js

```diff
@@ -40,7 +43,42 @@ class InventoryInput extends Component {
                                 <label className="dib b pr2">Currently Available:</label>
```

Comment:
I noticed you didn't include price and weight in InventoryEdit, was there a reason behind this?